components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,19 @@
+import { List } from './list';
+
+export default function ListComponent ({
+  component, eventHandlers
+}) {
```

Comment:
In one line 

```suggestion
export default function ListComponent ({ component, eventHandlers }) {
```

Also, did you setup eslint and editorconfig in your IDE?
https://github.com/Backendless/eslint-config
https://github.com/v-excelsior/editor-config/blob/main/.editorconfig